lib/index.js

```diff
@@ -152,18 +150,18 @@ internals.initialize = (server, next) => {
     Items.parallel(knexes, ping, (err) => {
 
         if (err) {
-            return next(err);
+            return err;
         }
 
         // Maybe run migrations
 
         if (!collector.migrateOnStart) {
-            return next();
+            return;
         }
 
         const rollback = (collector.migrateOnStart === 'rollback');
 
-        Migrator.migrate(collector.knexGroups, rootKnex, rollback, next);
+        Migrator.migrate(collector.knexGroups, rootKnex, rollback);
```

Comment:
Worth noting that dropping all these `next()` leaves the migration in a very intermediate/broken state– might be better off simply using the callback until it is written as an async function.

Reply:
ohhh yea. :) Definitely a "just get this fucking test to pass" change, done with 0 thought toward what it broke 🎺  sloppy sloppy. Thanks for catching, I'll have to dig more into that module 🙏